ISISNeutronMuon/MDANSE
# 3D view improvements

**Description of work**
Improves interaction between the GUI and the VTK 3D view.

closes #510

**Fixes**
1. Adds delegates to the atom properties table, preventing invalid values from being input.
2. Adds background colour selection.
3. Adds projection toggle button

**To test**
Load a trajectory and visualise it. Try changing all the options to see if any problems occur.
Please check also for atom selection dialog.
